conda_libmamba_solver/solver.py

```diff
@@ -47,6 +48,7 @@
 from .utils import CaptureStreamToFile
 
 log = logging.getLogger(f"conda.{__name__}")
+BLURB_COUNT = 0
```

Comment:
```suggestion
_BLURB_COUNT = False
```

Reply:
I played with both and have no preference either way 🤷‍♂️

Reply:
Let's leave it like it is then. No biggie.